### DISCUSSION TITLE
Unity Catalog Roadmap Discussions

### DISCUSSION THREAD

**Discussion by dennyglee** [unitycatalog/unitycatalog#294] [Q&A]
We had some great questions and discussions on the [Unity Catalog Slack](https://go.unitycatalog.io/slack) and the [Unity Catalog Community Meetup 2024-07-25](https://www.youtube.com/watch?v=bLPAN4_MGP8). If you have questions about [Milestone 0.2](https://github.com/unitycatalog/unitycatalog/milestone/2) or more, this is the discussion.

**Comment by ni-mi** [comment]
Hi @dennyglee 
Some questions:
1. I see the milestone is for September so I guess we don't decide on it's content as a community. When will we have such a community process?
2. What is the process to become a member of the community - and what are the possible roles and hierarchy?
3. Who will decide on UC roadmap - the community? Databricks? Databricks - by being the majority of the community :) ?
4. Will Databricks APIs match the APIs of the OSS UC? If so - is there a timetable for that as well?

Thanks!

**Reply by dennyglee** [reply]
Great questions @ni-mi , let me do my best to answer these questions
1. While we have [Milestone 0.2](https://github.com/unitycatalog/unitycatalog/milestone/2) targeting in September, we work together in this forum, [Slack](https://go.unitycatalog.io/slack), our bi-weekly UC community meetings ([rewatch](http://youtube.com/@unitycatalog-io)), and our upcoming contributor meetings.  It is through this process we will decide together what items will make it into the milestones and the cadence (e.g., do we release by specific features and/or release at a set schedule).

2. In terms of becoming a member of the Unity Catalog community, you already are :).  Saying this, some quick call outs:
* While we are LF AI & Data [sandbox project](https://lfaidata.foundation/projects/unity-catalog/) ([blog1](https://lfaidata.foundation/blog/2024/06/20/welcoming-unity-catalog-to-the-lf-ai-data-foundation-a-milestone-in-open-data-and-ai-governance/), [blog2](https://lfaidata.foundation/blog/2024/06/27/unity-catalog-first-meetup-recap-and-whats-next/)), we are still working on finalizing the *technical charter*.  
* The *technical charter* provides definitions such as *contributors*, *maintainers*, and *TSC*
* We're still working on those definitions and we will be sharing the proposed definitions as soon as this initial process is finalized.

3. The UC roadmap will be decided by the community. Once the *technical charter* we will have the governance structure around the voting process.  But as we are just staring out and delivering code, a lot of this is right now ad-hoc.   As of writing, there is a lot of excitement around integrations, Catalog UI, Rust client, and Spark integration (for example).  There are many folks volunteering their time to work on these topics from a wide variety of backgrounds and companies.  And thanks to everyone contributing! 

4. When I put on my *Databricks hat*, yes the goal is to have the Databricks APIs match the Unity Catalog APIs.  Saying this, Databricks discussions are typically going to be in Databricks forums (e.g., [Community](https://community.databricks.com), PABs, Slack, etc.).  In general, we will not be discussing Databricks here as the focus is how we can all make UC amazing for everyone.


HTH!
